app/helpers/application_helper.rb

```diff
@@ -293,7 +295,7 @@ def browse_name(args)
 
   def name_title(args)
     args[:document][args[:field]].each_with_index do |name_t, i|
-      next unless args[:document]['name_title_browse_s'] && args[:document]['name_title_browse_s'].include?(name_t)
+      next unless args[:document]['name_title_browse_s']&.include?(name_t)
```

Comment:
I hadn't seen that syntax before, but it seems like a nice way to make those checks less verbose.